lib/widgets/compose_box.dart

```diff
@@ -257,6 +317,91 @@ class _StreamSendButtonState extends State<_StreamSendButton> {
   }
 }
 
+class _AttachFileButton extends StatelessWidget {
```

Comment:
nit: let's put this before _StreamSendButton — that way things are in a bit more the logical order, in that sending is what you do at the end when you're done with attaching files and any other such operations